gulpfile.js

```diff
@@ -0,0 +1,16 @@
+var gulp    = require('gulp');
+var htmlmin = require('gulp-html-minifier');
+
+// Minify HTML
+gulp.task('minify', function() {
+  gulp.src('./src/github-btn.html')
+    .pipe(htmlmin({
+      collapseWhitespace: true,
+      minifyCSS: true,
+      minifyJS: true
```

Comment:
You need to add to remove optional tags in order to match the previous behavior.